<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotacionCientifica</class>
 <widget class="QDialog" name="NotacionCientifica">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>502</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>962</width>
    <height>502</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>962</width>
    <height>502</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NOTACIÓN CIENTIFÍCA</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="lb_mostrar_numero">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid  rgb(0, 0, 0);
text-align: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lE_base">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>310</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QSpinBox" name="sB_exponente">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>300</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>310</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;X10&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_volver">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>450</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid  rgb(255, 0, 0);
font-weight: bold;</string>
   </property>
   <property name="text">
    <string> VOLVER</string>
   </property>
   <property name="icon">
    <iconset resource="imagen.qrc">
     <normaloff>:/new/prefix1/recursos/icono_volver.png</normaloff>:/new/prefix1/recursos/icono_volver.png</iconset>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_random">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>305</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgb(0, 0, 0);
border-width: 2px;
border-radius: 10px;
color: rgb(0,0,0);
background-color: rgb(166, 166, 166);
font-weight: bold;</string>
   </property>
   <property name="text">
    <string> GENERAR NÚMERO
      ALEATORIO</string>
   </property>
   <property name="icon">
    <iconset resource="imagen.qrc">
     <normaloff>:/new/prefix1/recursos/icono_dado.png</normaloff>:/new/prefix1/recursos/icono_dado.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_calcular">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>310</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgb(0, 0, 0);
border-width: 2px;
border-radius: 10px;
color: rgb(0,0,0);
background-color: rgb(166, 166, 166);
font-weight: bold;</string>
   </property>
   <property name="text">
    <string>   VERIFICAR</string>
   </property>
   <property name="icon">
    <iconset resource="imagen.qrc">
     <normaloff>:/new/prefix1/recursos/icono_calcular.png</normaloff>:/new/prefix1/recursos/icono_calcular.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_revisar">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>380</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgb(0, 0, 0);
border-width: 2px;
border-radius: 10px;
color: rgb(0,0,0);
background-color: rgb(166, 166, 166);
font-weight: bold;</string>
   </property>
   <property name="text">
    <string>  REVISAR
RESPUESTA</string>
   </property>
   <property name="icon">
    <iconset resource="imagen.qrc">
     <normaloff>:/new/prefix1/recursos/icono_revisar.png</normaloff>:/new/prefix1/recursos/icono_revisar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>370</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;&amp;quot;Coloque en notación cientifíca el número generado&amp;quot;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="imagen.qrc"/>
 </resources>
 <connections/>
</ui>
